executable-spec/test/Cardano/Ledger/Spec/STS/Chain/Chain/Properties.hs

```diff
@@ -126,119 +171,102 @@ relevantCasesAreCovered
+          )
+          "a reasonable Pct of SIP submissions in Update Payload"
+          $
+        -- QC.tabulate "Pct of SIP submissions in Update Payload"
+        --                                              [( show @Int
+        --                                                $ round @Float
+        --                                                $ pctSIPsInUpdPayload traceSample
+        --                                               ) ++ "%"
+        --                                              ]
+        --    $
+        QC.cover 50
+          (pctSIPsTallyOutcome traceSample Data.Approved >= 5)
+          "satisfactory pct of approved SIPs"
+          $
+
+        QC.cover 1
+          (pctSIPsTallyOutcome traceSample Data.Rejected >= 0.01)
+          "satisfactory pct of rejected SIPs"
+          $
+
+        QC.cover 50
+          (pctSIPsTallyOutcome traceSample Data.Expired >= 0.01)
+          "satisfactory pct of expired SIPs"
+          $
+
+        QC.cover 1
+          (pctSIPsInRevoting traceSample Data.NoQuorum >= 1)
+          "satisfactory pct of SIPs in revoting NoQuorum"
+          $
+
+        QC.cover 50
+          (pctSIPsInRevoting traceSample Data.NoMajority >= 1)
+          "satisfactory pct of SIPs in revoting NoMajority"
+          $ True
+
+        -- QC.tabulate "Pct of SIP per Tally Outcome"
+        --                                   [
+                                            -- ( show @Int
+                                            --  $ round @Float
+                                            --  $ pctSIPsTallyOutcome traceSample Data.Approved
+                                            -- ) ++ "% Approved"
+                                          --  ( show @Int
+                                          --     $ round @Float
+                                          --     $ pctSIPsTallyOutcome traceSample Data.Rejected
+                                          --   ) ++ "% Rejected"
+                                          -- , ( show @Int
+                                          --     $ round @Float
+                                          --     $ pctSIPsTallyOutcome traceSample Data.NoQuorum
+                                          --   ) ++ "% NoQuorum"
+                                          -- , ( show @Int
+                                          --     $ round @Float
+                                          --     $ pctSIPsTallyOutcome traceSample Data.NoMajority
+                                          --   ) ++ "% NoMajority"
+                                          -- , ( show @Int
+                                          --     $ round @Float
+                                          --     $ pctSIPsTallyOutcome traceSample Data.Expired
+                                          --   ) ++ "% Expired"
+                                          -- ]
+           -- $
   where
-    maxTraceLength = 100
+    maxTraceLength = 200
```

Comment:
Hmmm, we shouldn't test with long traces unless there's a good reason for doing so.

Reply:
traces should be long enough to allow for the stabilization
of events

Reply:
The problem is not the trace length but the other parameters (like `k`, so we should choose `k` based on the trace length, and the same for all the other parameters (like vote duration periods)). 

If we don't address this here at least we should make an issue to change it. We should be very conservative about having long traces since this leads to longer testing time and bigger test space (which might make uncovering bugs hard).